src/views/teachers/landing/landing.jsx

```diff
@@ -85,7 +85,7 @@ var Landing = injectIntl(React.createClass({
                         <FlexRow className="guides-and-tutorials">
                             <div>
                                 <a href="/go">
-                                    <img src="/svgs/teachers/resources.svg" alt="resources icon" />
+                                    <img src="/svgs/teachers/cards.svg" alt="cards icon" />
```

Comment:
The new image doesn't match the size/aspect ration of the others so the text isn't aligned. You'll need to get a new version of the SVG from @carljbowman